charts/agent/templates/securitycontextconstraint.yaml

```diff
@@ -15,7 +15,8 @@ allowHostPID: true
 allowHostPorts: false
 allowPrivilegeEscalation: true
 allowPrivilegedContainer: true
-allowedCapabilities: []
+allowedCapabilities:
+{{- include "agent.capabilities" . }}
```

Comment:
Please make it conditional so we include the capabilities only when needed, keeping the empty list in other conditions.